get_staples.hh

```diff
@@ -30,3 +30,43 @@ template<class T, class S> void get_staples(T &K, gaugeconfig<S> &U,
   }
 }
 
+//stores c1*U1+complex(c2*U2) in U1
+//variable K used for accumulation is of type Complex for u1 (and su2 for su2?)
```

Comment:
its of type su2, but its no longer an SU(2) matrix, as you have guessed, I think.